### ISSUE TITLE
False positive with audit_rules_kernel_module_loading

### ISSUE THREAD

**Issue by pkbarbiedoll** [OpenSCAP/scap-security-guide#1022]
**OS**: 

```
cat /etc/redhat-release
Red Hat Enterprise Linux Server release 7.2 (Maipo)
(minimal install)
```

**OpenSCAP**:

```
Name        : openscap
Arch        : x86_64
Version     : 1.2.5
Release     : 3.el7
Size        : 49 M
Repo        : installed
From repo   : rhel-7-server-rpms
```

**SCAP Security Guide**:

```
Name        : scap-security-guide
Arch        : noarch
Version     : 0.1.25
Release     : 3.el7
Size        : 11 M
Repo        : installed
From repo   : rhel-7-server-rpms

```

**audit.rules:**

```
    # cat /etc/audit/rules.d/audit.rules 
        # snipped for brevity #  

    -w /sbin/insmod -p x -k modules
    -w /sbin/rmmod -p x -k modules
    -w /sbin/modprobe -p x -k modules
    -a always,exit -F arch=b64 -S init_module -S delete_module -k module
```

**Command used to generate report:**
`oscap oval eval --results $file_xml --report $file_html /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml`

**Results:**

> oval:ssg:def:656  **false**   compliance  [RHEL7_20150325], [FEDORA20_20150325], [**audit_rules_kernel_module_loading**]  **Audit Kernel Module Loading and Unloading**

**Expected Result:**

> oval:ssg:def:656  **true**    compliance  [RHEL7_20150325], [FEDORA20_20150325], [**audit_rules_kernel_module_loading**]  **Audit Kernel Module Loading and Unloading**

**Comment by iankko** [comment]
Thank you for the report. I will have a look @ this.

**Comment by shawndwells** [comment]
resolved sometime around 9b903461
